control-plane/pkg/reconciler/testing/objects_broker.go

```diff
@@ -219,6 +219,12 @@ func WithAutoscalingAnnotationsTrigger() reconcilertesting.TriggerOption {
 
 type CMOption func(cm *corev1.ConfigMap)
 
+func WithConfigMapNamespace(namespace string) CMOption {
```

Comment:
Should we reflect the `namespaced` in the name? (as this is for the namespaced broker)?

Reply:
This function is a mutator for a Configmap. It just sets the namespace of a Configmap.

It can be used in any tests.